lib/containers/k8s.pm

```diff
@@ -27,19 +27,25 @@ Installs k3s, checks the instalation and prepare the kube/config
 =cut
 
 sub install_k3s {
-    zypper_call('in apparmor-parser') if (is_sle('>=15-sp1'));
+    if (get_var("DISTRI") eq "alp") {
+        assert_script_run('/usr/bin/k3s-install');
+    }
+    else {
+        zypper_call('in apparmor-parser') if (is_sle('>=15-sp1'));
   # Apply additional options. For more information see https://rancher.com/docs/k3s/latest/en/installation/install-options/#options-for-installation-with-script
-    my $k3s_version = get_var("CONTAINERS_K3S_VERSION");
-    if ($k3s_version) {
-        record_info('k3s', $k3s_version);
-        assert_script_run("export INSTALL_K3S_VERSION=$k3s_version");
+        my $k3s_version = get_var("CONTAINERS_K3S_VERSION");
+        if ($k3s_version) {
+            record_info('k3s', $k3s_version);
+            assert_script_run("export INSTALL_K3S_VERSION=$k3s_version");
+        }
+        assert_script_run("export INSTALL_K3S_SYMLINK=" . get_var('K3S_SYMLINK')) if (get_var('K3S_SYMLINK'));
+        assert_script_run("export INSTALL_K3S_BIN_DIR=" . get_var('K3S_BIN_DIR')) if (get_var('K3S_BIN_DIR'));
+        assert_script_run("export INSTALL_K3S_CHANNEL=" . get_var('K3S_CHANNEL')) if (get_var('K3S_CHANNEL'));
```

Comment:
I don't understand those. Can you add some comment?

Reply:
Sorry but this is something that came from a previous commit and isn't used here. We could create a ticket to document that

Reply:
@pdostal See the comment 8 lines before :slightly_smiling_face: